README.md

```diff
@@ -9,9 +11,34 @@ Full featured clan system for PVP Minecraft Servers!
 [![Crowdin](https://badges.crowdin.net/simpleclans/localized.svg)](https://crowdin.com/project/simpleclans)
 [![Discord](https://img.shields.io/discord/719557355917934613?label=discord&logo=discord)](https://discord.gg/CkNwgdE)
 
-Spigot resource page: https://www.spigotmc.org/resources/simpleclans.71242/
+#### Download Link
+
+* [Download from Spigot](https://www.spigotmc.org/resources/simpleclans.71242/)
+
+#### Documentation 
 
-Wiki page: https://github.com/RoinujNosde/SimpleClans/wiki
+* [Documentation](https://simpleclans.gitbook.io/simpleclans/)
+* Commands & Permissions
+  * [Permissions](https://simpleclans.gitbook.io/simpleclans/commands-and-permissions/permissions)
+  * [Commands](https://simpleclans.gitbook.io/simpleclans/commands-and-permissions/commands)
+  * [Clan Alliances and Rivalries](https://simpleclans.gitbook.io/simpleclans/commands-and-permissions/aliances-and-rivalries)
+  * [Clan Ranks with Permissions](https://simpleclans.gitbook.io/simpleclans/commands-and-permissions/ranks-with-permissions)
+* How to Setup
+  * [Configuration](https://simpleclans.gitbook.io/simpleclans/how-to-setup/configuration)
+  * [Translation](https://simpleclans.gitbook.io/simpleclans/how-to-setup/translation)
+  * [Clan Member Fee](https://simpleclans.gitbook.io/simpleclans/how-to-setup/member-fee)
+  * [Clan Upkeep](https://simpleclans.gitbook.io/simpleclans/how-to-setup/clan-upkeep)
+  * [Clan name above players head](https://simpleclans.gitbook.io/simpleclans/how-to-setup/clan-below-players-name)
+  * [Clan name on Tablist](https://simpleclans.gitbook.io/simpleclans/how-to-setup/clan-on-tablist)
+* Developers
```

Comment:
Just noticed this.
PAPI and Land plugins are not for devs.
This title could be "Other".

Reply:
I changed it to "plugins & development" as other isn't very descriptive.

Reply:
Awesome